src/main/java/org/apache/rocketmq/exporter/config/RMQConfigure.java

```diff
@@ -44,6 +44,16 @@
 
     private volatile String rocketmqVersion = System.getProperty(ROCKETMQ_CONFIG_ROCKETMQ_VERSION, "V4_3_2");
 
+    private String accessKey;
+
+    private String secretKey;
+
+    public boolean enableACL() {
+        return !(StringUtils.isAllBlank(this.accessKey)
+                &&
+                StringUtils.isAllBlank(this.secretKey));
+    }
+
```

Comment:
I think it would be better to make it into a configurable switch for enableACL rather than by whether AK/SK has a value.

Reply:
LGTM and updated